HW02_B_Mimic_starter.Rmd

```diff
@@ -36,13 +36,21 @@ Try for all 4, but if you are struggling don't worry about it. Try your best for
 ```{r, echo = T, include = TRUE}
 data("diamonds")
 #hint think about the *position* the bars are in...
+
 ```
 
 
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}
-
-
+ggplot(diamonds, aes(x = cut, fill = clarity)) +
+  geom_bar(position = "dodge") +
+  labs(title = "My Diamon Collection", 
+       subtitle = "Boxplot representing the number of diamonds in my diamond collection by\ntype of cut quality and clarity of diamond", 
+       x = "Diamond Cut", 
+       y = "Number of Diamonds") +
+  theme(plot.title = element_text(hjust = 0.53)) +
+    annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") +
+  annotate("rect", xmin = 4.5, ymin = 0, xmax = 5.5, ymax = 5000, fill = "grey40", alpha = 0.2)

```

Comment:
The code looks great, and the graph is basically identical to the one provided in the assignment. One thing that could be useful in general would be to add some comments to specific lines in the code, indicating what they do. I need to work on this as well actually, but I think it can be very helpful for understanding and following the code.